src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/configure.jelly

```diff
@@ -45,27 +45,64 @@ THE SOFTWARE.
 
         <st:include page="configure-details" optional="true"/>
 
-        <j:if test="${it.views.size()>1}">
-          <f:entry title="${%Default View}" field="defaultView">
-            <select class="setting-input" name="primaryView">
-              <j:forEach var="v" items="${it.views}">
+        <j:choose>
```

Comment:
Each concrete folder type only ever allows one view holder... I'd need to make it an `AbstractDescribable` if I wanted to move it to there... and then we end up with a drop down selector that I need to force read-only... just as messy when I tried it